health-probe-proxy/example.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
```

Comment:
let's add the deployment charts into helm charts as part of cnm deployment (a new sidecar for it). And please add a new helm config to enable this sidecar, which is disable by default during preview.

Reply:
done